frontend/express/public/javascripts/countly/vue/templates/sidebar/analytics-menu.html

```diff
@@ -24,8 +25,21 @@
                     </template>
                 </el-input>
             </template>
-        </cly-select-x>
+        </cly-select-x> -->
+        <div class="cly-vue-sidebar__app-image" :style="{backgroundImage: 'url(' + activeApp.image + ')'}"></div>
+        <div>
+            <div class="cly-vue-sidebar__app-name-type bu-is-flex bu-is-flex-direction-column bu-ml-1">
+                <span class="cly-vue-sidebar__app-name">{{activeApp.name}}</span>
+                <span class="cly-vue-sidebar__app-type font-italic"> {{activeApp.type}}</span>
+            </div>
+        </div>
+        <span class="el-input__suffix">
+            <span class="el-input__suffix-inner">
+                <i  @click = "toggleAppSelection()"class="el-select__caret el-input__icon el-icon-arrow-down"></i>
```

Comment:
why do you call **toggleAppSelection** on clicking the icon?

Reply:
Hover ?